src/App.tsx

```diff
@@ -2,108 +2,41 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+
+import { User } from './types/User';
+import { Comment } from './types/Comment';
+import { Post } from './types/Post';
+import { Fullpost } from './types/Fullpost';
+import { PostList } from './components/PostList';
+
+function getAuthorById(authors: User[], postId: number) {
```

Comment:
You don't need to path users as an argument. It's available from the outer scope.

Reply:
Fixed